code/modules/mod/mod_ai.dm

```diff
@@ -86,3 +87,49 @@
 #undef WEARER_DELAY
 #undef LONE_DELAY
 #undef CELL_PER_STEP
+
+/obj/item/mod/control/proc/ai_fall()
+	if(!wearer)
+		return
+	REMOVE_TRAIT(wearer, TRAIT_FORCED_STANDING, MOD_TRAIT)
+
+/obj/item/mod/ai_minicard
+	name = "AI mini-card"
+	desc = "A small card designed to eject dead AIs. You could use an intellicard to recover it."
+	icon = 'icons/obj/aicards.dmi'
+	icon_state = "minicard"
+	var/mob/living/silicon/ai/stored_ai
```

Comment:
QDEL_NULL (since I think this is an ownership contract)

Also, make this a weakref in case the AI deletes